cjworkbench/i18n/trans.py

```diff
@@ -28,38 +28,35 @@ def _get_translations(locale):
     return _translators[locale]
 
 
-def trans(message_id, *, default, context=None, parameters={}):
-    """Translate the given message ID to the current locale
-    
-    HTML is not escaped.
+def trans(message_id, *, default, parameters={}):
+    """Mark a message for translation and localize it to the current locale, not escaping HTML.
```

Comment:
Let's nix "not escaping HTML" from this docstring. Mention of "HTML" makes sense when reading this file and when writing unit tests, but that's where its utility stops. For callers, I think it's more likely to mislead than to help.

(The mention of HTML implies that callers should _consider_ HTML-escaping, more than they would consider this with other string functions. But you don't see "not escaping HTML" in the `str.join()` docstring, for instance. And even if we started using `trans()` within HTML templates, one could simply write `{% trans xxx, xxx, xxx %}` without thinking of HTML escaping, just like with any other function.